apps/web/src/features/base/invitation/components/RemoveBaseMemberButton.tsx

```diff
@@ -34,7 +34,6 @@ const RemoveBaseMemberButton = ({
         memberId: member.member.id,
       })
       router.refresh()
-      close()
```

Comment:
Modal is null when member has not accepted the invitation, so it triggered a "Cannot read properties of null (reading 'modal')"

<img width="686" height="306" alt="image" src="https://github.com/user-attachments/assets/5530c8eb-9167-45c0-8891-7cdf8f1df204" />

Reply:
(and also it close the modal even without the close() call)